apps/webapp/shared/responsive-image.tsx

```diff
@@ -0,0 +1,17 @@
+// https://nextjs.org/docs/messages/next-image-upgrade-to-13
+import Image from 'next/legacy/image';
```

Comment:
Why are we using the legacy image? We should be using the latest.

Reply:
I had an error related to this - https://nextjs.org/docs/messages/next-image-upgrade-to-13

Reply:
but there is new example - https://nextjs.org/docs/app/building-your-application/optimizing/images#fill-container! i`ll add this example